doc/tutorial/image/tutorial-basic-drawings.doc

```diff
@@ -0,0 +1,112 @@
+
+\snippet tutorial-viewer.cpp vpImage construction
+Create an instance of a color image where each pixel is coded in RGBa.
+
+\snippet tutorial-viewer.cpp vpImage reading
+The image \c I is initialized by reading an image file from the disk. If the image format is not supported we throw an exception.
+
+\snippet tutorial-viewer.cpp vpDisplay construction
+Create an instance of an image display window for image \c I. The first viewer that is available is used. Here we create the link between the image \c I and the display \c d. Note that an image can only have one display. 
+
+\snippet tutorial-viewer.cpp vpDisplay set title
+The title of the display is then set to \c "My image".  
+
+\snippet tutorial-viewer.cpp vpDisplay display
+First we display the content of the image \c I, then we flush the display to render the image.
+
+\snippet tutorial-viewer.cpp vpDisplay get click
+Here we handle mouse events. We are waiting for a blocking mouse click to end the program.
+
+Here is a screen shot of the resulting output window :
+
+  \image html img-monkey.png
+
+\section basic_drawings How to draw basic drawings
+
+\subsection point How to draw a point
+
+We use displayPoint function of vpDisplay class to draw a point on the screen.
+
+\include tutorial-draw-point.cpp
+
+Here we draw a point at the center of a grey image with red color and thickness 2.
+
+\subsection line How to draw a line segment
+
+We use displayLine function of vpDisplay class to draw a line segment on the screen.
+
+\snippet tutorial-draw-line.cpp Line
+
+Here we draw a red coloured line segment with the specified initial and final coordinates and thickness 10.
+
+\subsection circle How to draw a circle
+
+We use displayCircle function of vpDisplay class to draw a circle on the screen.
+
+\snippet tutorial-image-display-scaled-auto.cpp Circle
+
+Here we draw a red coloured filled circle at the center with radius of 200.
+
+\subsection rectangle How to draw a rectangle
+
+We use displayRectangle function of vpDisplay class to draw a rectangle on the screen.
+
+\snippet tutorial-draw-rectangle.cpp Rectangle
+
+Here we draw a red coloured filled rectangle with specified top-left coordinates and width and height.
+
+\subsection cross How to draw a cross
+
+We use displayCross function of vpDisplay class to draw a rectangle on the screen.
+
+\snippet tutorial-draw-cross.cpp Cross
+
+Here we draw a red coloured corss on the center with speicfied size and thickness 2.
+
+\section adding_text How to add text in the window overlay
+
+We use displayCharString of vpDisplay class to add text in the window overlay.
+
+\include tutorial-draw-text.cpp
+
+\section adding_text How to export an image
+
+We use getImage of vpDisplay class to update the image with an overlay. 
+
+\include tutorial-export-image.cpp
+
+\section event_keyboard How to handle keyboard events
+
+We use getKeyboardEvent of vpDisplay class to get the value of the key pressed.
+
+\include tutorial-event-keyboard.cpp
```

Comment:
- Add an explanation: checking 5 key events before exiting the while loop...
- There should be a "Next tutorial" section. D'ont forget to close the doxygen comment with `*/`
```
\section basic_drawings_next Next tutorial

You are now ready to see the \ref tutorial-grabber or \ref tutorial-image-filtering.

*/
```

- Modify the "Next tutorial" section in `tutorial-image-display.doc` to point to `tutorial-basic-drawings` with something like "You are now ready to follow the \ref tutorial-basic-drawings."